Cargo.toml

```diff
@@ -13,7 +15,13 @@ reqwest = { version = "0.11", features = ["json"] }
 chrono = { version = "0.4", features = ["serde"] }
 either = "1.8"
 
-tokio = { version = "1.17", features = ["macros", "sync", "fs", "io-util", "rt-multi-thread"] }
+tokio = { version = "1.17", features = [
+    "macros",
```

Comment:
Tbh I don't really like this change (nor the one above). Would it be possible to not change the `aws-config` and `tokio` lines? I know, it's formatting, and ultimately doesn't matter, but on the other hand I don't want to approve it if it is not added to CI and enforced consistently / configured at a central place.

`git checkout -p origin/main Cargo.toml`

Reply:
I totally rely on linters usually -- for most things my editor just auto-formats on save.  I'm not sure there's a standard one for toml though.  If we add one, all this will just pop back to whatever the linter's preference is (which might be the same as the vscode plugin that I used here), no harm done.

Reply:
FTR there is ongoing discussion in the rustfmt repo to also format Cargo.toml: https://github.com/rust-lang/rustfmt/pull/5240